executor/ddl.go

```diff
@@ -350,8 +350,19 @@ func (e *DDLExec) executeDropSequence(s *ast.DropSequenceStmt) error {
 
 // dropTableObject actually applies to `tableObject`, `viewObject` and `sequenceObject`.
 func (e *DDLExec) dropTableObject(objects []*ast.TableName, obt objectType, ifExists bool) error {
-	var notExistTables []string
+	var (
```

Comment:
Dropping a local temporary table should not commit current transaction implicitly, it's better to make it done in this pr and add some tests.

For users can drop multi tables in one statement, behavior should also be determined if both temporary tables and normal tables are included in the table list.